.github/workflows/requirements/docs.txt

```diff
@@ -5,6 +5,9 @@ codespell==2.4.1
 pandas==2.2.3
 pyyaml==6.0.2
 sphinxcontrib-programoutput==0.18
+# benchpark analyze -h command
```

Comment:
How can we make these dependencies optional? @michaelmckinsey1

Reply:
They are optional, this is for the docs output only. For users, they would need to run `pip install .[analyze]` from the benchpark directory to install these dependencies, which is in the docs.